test/index.js

```diff
@@ -61,9 +61,15 @@ describe('doxray.js, core methods', function() {
       );
     });
 
-    it('should respect custom regex patterns in options', function () {
+    it('should respect custom regex patterns in options while retaining defaults', function () {
```

Comment:
I'm not sure I get how this test is checking to see if the defaults are retained.

To do that I would want to remove the HTML key from this options object and then parse an HTML file to see if it works. If it does then presumably it's using the HTML regex from Doxray.prototype. If it doesn't then it might mean that the regex option is replacing the prototype.

Reply:
Yeah, I was moving fast and misread both these tests. Should be better now?